serverless.yaml

```diff
@@ -630,7 +630,7 @@ resources:
               - id: W35
                 reason: 'This is the logs bucket for access logs'
         Properties:
-          AccessControl: LogDeliveryWrite
+          #AccessControl: LogDeliveryWrite
```

Comment:
Deployment works, so I guess this configuration is fine. But just a quick look at the docu, it looks like this defaults to `Private`, which means `Owner gets FULL_CONTROL. No one else has access rights (default).` https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#canned-acl

I'm not sure what this means in the context of this `ObjectOwnership: ObjectWriter` config. Does this mean objects can be written to the bucket by entities besides the owner of the bucket, but the owner of the bucket can access these objects?

Reply:
I took over the settings from latest FhirWorks repo state, see https://github.com/aws-solutions/fhir-works-on-aws/blob/main/solutions/deployment/lib/cdk-infra-stack.ts. Starting at line 258
`
  const fhirBinaryBucket = new Bucket(this, 'fhirBinaryBucket', {
      serverAccessLogsBucket: fhirLogsBucket,
      serverAccessLogsPrefix: 'binary-acl',
      versioned: true,
      encryption: BucketEncryption.KMS,
      encryptionKey: kmsResources.s3KMSKey,
      removalPolicy: RemovalPolicy.RETAIN,
      blockPublicAccess: {
        blockPublicAcls: true,
        blockPublicPolicy: true,
        ignorePublicAcls: true,
        restrictPublicBuckets: true
      },
      objectOwnership: ObjectOwnership.OBJECT_WRITER
    });
``